.github/workflows/avm.res.digital-twins.digital-twins-instance.yml

```
@@ -82,7 +82,9 @@ jobs:
     uses: ./.github/workflows/avm.template.module.yml
     with:
       workflowInput: "${{ needs.job_initialize_pipeline.outputs.workflowInput }}"
-      moduleTestFilePaths: "${{ needs.job_initialize_pipeline.outputs.moduleTestFilePaths }}"
+      # moduleTestFilePaths: "${{ needs.job_initialize_pipeline.outputs.moduleTestFilePaths }}"
+      moduleTestFilePaths: "[{\"path\":\"tests/e2e/max/main.test.bicep\",\"name\":\"max\"}]"
+      # moduleTestFilePaths: "[{\"path\":\"tests/e2e/testMe/main.test.bicep\",\"name\":\"testMe\"}]"
```

```suggestion
      moduleTestFilePaths: "${{ needs.job_initialize_pipeline.outputs.moduleTestFilePaths }}"
```
once done with the PR...